terraform/github/repositories/data.tf

```
@@ -10,4 +10,8 @@ data "github_team" "operations_engineering_test" {
 data "github_team" "test_team_access" {
   provider = github.ministryofjustice-test
   slug     = "test-team-access"
-}
\ No newline at end of file
+}
+
+data "github_team" "cloud-optimisation-and-accountability" {
```

```suggestion
data "github_team" "cloud_optimisation_and_accountability" {
```
Going by the others I think this may need to be underscores, the slug seems happy with hyphens...